Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1019,6 +1010,25 @@ def update_remote_system_command(client, args):
         return remote_args.remote_incident_id
 
 
+def update_related_alerts(client: Client, args: dict):
+    new_status = args.get('status')
+    incident_id = args.get('incident_id')
+    comment = f"resolved by XSOAR, due to incident {incident_id} has been resolved"
+    demisto.debug(f"{new_status=}, {comment=}")
+    if not new_status:
+        raise DemistoException(f"Failed to update alerts related to incident {incident_id},"
+                               "no status found")
+    if new_status == 'resolved_duplicate_incident':
+        new_status = 'resolved_duplicate'
+    incident_extra_data = client.get_incident_extra_data(incident_id)
+    if 'alerts' in incident_extra_data and 'data' in incident_extra_data['alerts']:
+        alerts_array = incident_extra_data['alerts']['data']
+        related_alerts_ids_array = [str(alert['alert_id']) for alert in alerts_array if 'alert_id' in alert]
+        demisto.debug(f"{related_alerts_ids_array=}")
+        args_for_command = {'alert_ids': related_alerts_ids_array, 'status': new_status, 'comment': comment}
+        return_results(update_alerts_in_xdr_command(client, args_for_command))
```

Comment:
```suggestion
        update_alerts_in_xdr_command(client, args_for_command)
```
Is it needed?

Reply:
I deleted it and it works without

Reply:
Yes, unless it can not be shown in the war room.